### PR TITLE
add some German translations

### PR DESCRIPTION
**Pull request by hashfunktion**
add some German translations

### CODE REVIEW

File: Translations/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -217,18 +217,18 @@ Note that the short string is limited to 30 characters and the long string is li
 "Monthly sip of coffee" = "Monthly sip of coffee";
 "Gift Keka a sip of coffee each month" = "Gift Keka a sip of coffee each month";
 "Monthly coffee" = "Monthly coffee";
```
Comment (aonez) [inline Translations/de.lproj/Localizable.strings:219]: Can you translate those too?

File: Translations/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -217,18 +217,18 @@ Note that the short string is limited to 30 characters and the long string is li
...
-"Tip Keka a dinner so it keeps rolling" = "Tip Keka a dinner so it keeps rolling";
-"Tip a bunch of food" = "Tip a bunch of food";
-"Tip Keka a new bunch of food to keep rolling" = "Tip Keka a new bunch of food to keep rolling";
+"Gift Keka a coffee each month" = "Schenke Keka jeden Monat einen Kaffee";
+"Monthly pair of coffees" = "Monatlich ein Paar Kaffee";
+"Gift Keka a couple coffees each month" = "Schenke Keka monatlich ein Paar Kaffee";
+"Monthly breakfast" = "Monatliches Frühstück";
+"Gift Keka a breakfast each month" = "Schenke Keka jeden Monat ein Frühstück";
+"Tip a coffee" = "Gebe Trinkgeld für einen Kaffee";
+"Tip Keka a coffee so it can stay awake" = "Gebe Keka Trinkgeld für einen Kaffee, damit es wach bleibt";
```
Comment (aonez) [inline Translations/de.lproj/Localizable.strings:226]: This is limited to 45 characters

File: Translations/de.lproj/Localizable.strings [hunk 3]
```diff
@@ -217,18 +217,18 @@ Note that the short string is limited to 30 characters and the long string is li
...
-"Tip a dinner" = "Tip a dinner";
-"Tip Keka a dinner so it keeps rolling" = "Tip Keka a dinner so it keeps rolling";
-"Tip a bunch of food" = "Tip a bunch of food";
-"Tip Keka a new bunch of food to keep rolling" = "Tip Keka a new bunch of food to keep rolling";
+"Gift Keka a coffee each month" = "Schenke Keka jeden Monat einen Kaffee";
+"Monthly pair of coffees" = "Monatlich ein Paar Kaffee";
+"Gift Keka a couple coffees each month" = "Schenke Keka monatlich ein Paar Kaffee";
+"Monthly breakfast" = "Monatliches Frühstück";
+"Gift Keka a breakfast each month" = "Schenke Keka jeden Monat ein Frühstück";
+"Tip a coffee" = "Gebe Trinkgeld für einen Kaffee";
```
Comment (aonez) [inline Translations/de.lproj/Localizable.strings:225]: This is limited to 30 characters

File: Translations/de.lproj/Localizable.strings [hunk 4]
```diff
@@ -217,18 +217,18 @@ Note that the short string is limited to 30 characters and the long string is li
...
+"Tip Keka a coffee so it can stay awake" = "Gebe Keka Trinkgeld für einen Kaffee, damit es wach bleibt";
+"Tip a breakfast" = "Gebe Trinkgeld für ein Frühstück";
```
Comment (aonez) [inline Translations/de.lproj/Localizable.strings:227]: This is limited to 30 characters

File: Translations/de.lproj/Localizable.strings [hunk 5]
```diff
@@ -217,18 +217,18 @@ Note that the short string is limited to 30 characters and the long string is li
...
+"Tip Keka a breakfast so it can start fresh" = "Gebe Keka Trinkgeld für ein Frühstück, damit es frisch starten kann";
+"Tip a lunch" = "Gebe Trinkgeld für ein Mittagessen";
```
Comment (aonez) [inline Translations/de.lproj/Localizable.strings:229]: This is limited to 30 characters

File: Translations/de.lproj/Localizable.strings [hunk 6]
```diff
@@ -217,18 +217,18 @@ Note that the short string is limited to 30 characters and the long string is li
...
+"Tip Keka a lunch and feed it" = "Gebe Keka Trinkgeld für ein Mittagessen und füttere es";
+"Tip a dinner" = "Gebe Trinkgeld für ein. Abendessen";
+"Tip Keka a dinner so it keeps rolling" = "Gebe Keka Trinkgeld für ein Abendessen und halte es am laufen";
+"Tip a bunch of food" = "Gebe Trinkgeld für ein Bündel essen";
```
Comment (aonez) [inline Translations/de.lproj/Localizable.strings:233]: This is limited to 30 characters

File: Translations/de.lproj/Localizable.strings [hunk 7]
```diff
@@ -217,18 +217,18 @@ Note that the short string is limited to 30 characters and the long string is li
...
+"Gift Keka a couple coffees each month" = "Schenke Keka monatlich ein Paar Kaffee";
+"Monthly breakfast" = "Monatliches Frühstück";
+"Gift Keka a breakfast each month" = "Schenke Keka jeden Monat ein Frühstück";
+"Tip a coffee" = "Gebe Trinkgeld für einen Kaffee";
+"Tip Keka a coffee so it can stay awake" = "Gebe Keka Trinkgeld für einen Kaffee, damit es wach bleibt";
+"Tip a breakfast" = "Gebe Trinkgeld für ein Frühstück";
+"Tip Keka a breakfast so it can start fresh" = "Gebe Keka Trinkgeld für ein Frühstück, damit es frisch starten kann";
+"Tip a lunch" = "Gebe Trinkgeld für ein Mittagessen";
+"Tip Keka a lunch and feed it" = "Gebe Keka Trinkgeld für ein Mittagessen und füttere es";
+"Tip a dinner" = "Gebe Trinkgeld für ein. Abendessen";
```
Comment (aonez) [inline Translations/de.lproj/Localizable.strings:231]: This is limited to 30 characters

File: Translations/de.lproj/Localizable.strings [hunk 8]
```diff
@@ -217,18 +217,18 @@ Note that the short string is limited to 30 characters and the long string is li
...
+"Tip Keka a dinner so it keeps rolling" = "Gebe Keka Trinkgeld für ein Abendessen und halte es am laufen";
+"Tip a bunch of food" = "Gebe Trinkgeld für ein Bündel essen";
+"Tip Keka a new bunch of food to keep rolling" = "Gebe Keka Trinkgeld für ein neues Bündel essen und halte es ma laufen";
```
Comment (aonez) [inline Translations/de.lproj/Localizable.strings:234]: This is limited to 45 characters

File: Translations/de.lproj/Localizable.strings [hunk 9]
```diff
@@ -217,18 +217,18 @@ Note that the short string is limited to 30 characters and the long string is li
...
+"Monthly breakfast" = "Monatliches Frühstück";
+"Gift Keka a breakfast each month" = "Schenke Keka jeden Monat ein Frühstück";
+"Tip a coffee" = "Gebe Trinkgeld für einen Kaffee";
+"Tip Keka a coffee so it can stay awake" = "Gebe Keka Trinkgeld für einen Kaffee, damit es wach bleibt";
+"Tip a breakfast" = "Gebe Trinkgeld für ein Frühstück";
+"Tip Keka a breakfast so it can start fresh" = "Gebe Keka Trinkgeld für ein Frühstück, damit es frisch starten kann";
+"Tip a lunch" = "Gebe Trinkgeld für ein Mittagessen";
+"Tip Keka a lunch and feed it" = "Gebe Keka Trinkgeld für ein Mittagessen und füttere es";
+"Tip a dinner" = "Gebe Trinkgeld für ein. Abendessen";
+"Tip Keka a dinner so it keeps rolling" = "Gebe Keka Trinkgeld für ein Abendessen und halte es am laufen";
```
Comment (aonez) [inline Translations/de.lproj/Localizable.strings:232]: This is limited to 45 characters

File: Translations/de.lproj/Localizable.strings [hunk 10]
```diff
@@ -217,18 +217,18 @@ Note that the short string is limited to 30 characters and the long string is li
...
+"Monthly pair of coffees" = "Monatlich ein Paar Kaffee";
+"Gift Keka a couple coffees each month" = "Schenke Keka monatlich ein Paar Kaffee";
+"Monthly breakfast" = "Monatliches Frühstück";
+"Gift Keka a breakfast each month" = "Schenke Keka jeden Monat ein Frühstück";
+"Tip a coffee" = "Gebe Trinkgeld für einen Kaffee";
+"Tip Keka a coffee so it can stay awake" = "Gebe Keka Trinkgeld für einen Kaffee, damit es wach bleibt";
+"Tip a breakfast" = "Gebe Trinkgeld für ein Frühstück";
+"Tip Keka a breakfast so it can start fresh" = "Gebe Keka Trinkgeld für ein Frühstück, damit es frisch starten kann";
+"Tip a lunch" = "Gebe Trinkgeld für ein Mittagessen";
+"Tip Keka a lunch and feed it" = "Gebe Keka Trinkgeld für ein Mittagessen und füttere es";
```
Comment (aonez) [inline Translations/de.lproj/Localizable.strings:230]: This is limited to 45 characters

File: Translations/de.lproj/Localizable.strings [hunk 11]
```diff
@@ -217,18 +217,18 @@ Note that the short string is limited to 30 characters and the long string is li
...
-"Tip Keka a new bunch of food to keep rolling" = "Tip Keka a new bunch of food to keep rolling";
+"Gift Keka a coffee each month" = "Schenke Keka jeden Monat einen Kaffee";
+"Monthly pair of coffees" = "Monatlich ein Paar Kaffee";
+"Gift Keka a couple coffees each month" = "Schenke Keka monatlich ein Paar Kaffee";
+"Monthly breakfast" = "Monatliches Frühstück";
+"Gift Keka a breakfast each month" = "Schenke Keka jeden Monat ein Frühstück";
+"Tip a coffee" = "Gebe Trinkgeld für einen Kaffee";
+"Tip Keka a coffee so it can stay awake" = "Gebe Keka Trinkgeld für einen Kaffee, damit es wach bleibt";
+"Tip a breakfast" = "Gebe Trinkgeld für ein Frühstück";
+"Tip Keka a breakfast so it can start fresh" = "Gebe Keka Trinkgeld für ein Frühstück, damit es frisch starten kann";
```
Comment (aonez) [inline Translations/de.lproj/Localizable.strings:228]: This is limited to 45 characters

**Review by aonez — CHANGES_REQUESTED** [review]
Thanks a lot @hashfunktion! Can you check the issues with the gift strings?

**Comment by hashfunktion** [comment]
Thanks for the tip.
I had skipped over the limitations, so now it has to be within the limits.

File: Translations/de.lproj/Localizable.strings (hunk 11 above)
Comment (hashfunktion) [reply Translations/de.lproj/Localizable.strings]: I corrected that

File: Translations/de.lproj/Localizable.strings (hunk 10 above)
Comment (hashfunktion) [reply Translations/de.lproj/Localizable.strings]: I corrected that

File: Translations/de.lproj/Localizable.strings (hunk 9 above)
Comment (hashfunktion) [reply Translations/de.lproj/Localizable.strings]: I corrected that

File: Translations/de.lproj/Localizable.strings (hunk 8 above)
Comment (hashfunktion) [reply Translations/de.lproj/Localizable.strings]: I corrected that

File: Translations/de.lproj/Localizable.strings (hunk 7 above)
Comment (hashfunktion) [reply Translations/de.lproj/Localizable.strings]: I corrected that

File: Translations/de.lproj/Localizable.strings (hunk 6 above)
Comment (hashfunktion) [reply Translations/de.lproj/Localizable.strings]: I corrected that

File: Translations/de.lproj/Localizable.strings (hunk 5 above)
Comment (hashfunktion) [reply Translations/de.lproj/Localizable.strings]: I corrected that

File: Translations/de.lproj/Localizable.strings (hunk 4 above)
Comment (hashfunktion) [reply Translations/de.lproj/Localizable.strings]: I corrected that

File: Translations/de.lproj/Localizable.strings (hunk 3 above)
Comment (hashfunktion) [reply Translations/de.lproj/Localizable.strings:225]: I corrected that

File: Translations/de.lproj/Localizable.strings (hunk 2 above)
Comment (hashfunktion) [reply Translations/de.lproj/Localizable.strings]: I corrected that

File: Translations/de.lproj/Localizable.strings (hunk 1 above)
Comment (hashfunktion) [reply Translations/de.lproj/Localizable.strings]: Sure, I have translate those in the new commit

File: Translations/de.lproj/Localizable.strings [hunk 12]
```diff
@@ -215,20 +215,20 @@ The language used in MAS/AppleID will be used in Keka's tip strings and may not
...
-"Tip a bunch of food" = "Tip a bunch of food";
-"Tip Keka a new bunch of food to keep rolling" = "Tip Keka a new bunch of food to keep rolling";
+"Gift Keka a sip of coffee each month" = "Schenke Keka jeden Monat einen Schluck Kaffee";
+"Monthly coffee" = "Monatlicher Kaffee";
+"Gift Keka a coffee each month" = "Schenke Keka jeden Monat einen Kaffee";
+"Monthly pair of coffees" = "Monatlich ein Paar Kaffee";
+"Gift Keka a couple coffees each month" = "Schenke Keka monatlich ein Paar Kaffee";
+"Monthly breakfast" = "Monatliches Frühstück";
+"Gift Keka a breakfast each month" = "Schenke Keka jeden Monat ein Frühstück";
+"Tip a coffee" = "Gebe Trinkgeld für einen Kaffee";
```
Comment (aonez) [inline Translations/de.lproj/Localizable.strings:225]: This one still is more than 30 characters. Maybe "Gebe Trinkgeld für ein Kaffee"?

File: Translations/de.lproj/Localizable.strings [hunk 13]
```diff
@@ -215,20 +215,20 @@ The language used in MAS/AppleID will be used in Keka's tip strings and may not
 Note that the short string is limited to 30 characters and the long string is limited to 45 characters.
 */
 "Monthly sip of coffee" = "Monthly sip of coffee";
```
Comment (aonez) [inline Translations/de.lproj/Localizable.strings:217]: This one is missing if you can also translate it :)

**Review by aonez — CHANGES_REQUESTED** [review]

File: Translations/de.lproj/Localizable.strings (hunk 13 above)
Comment (hashfunktion) [reply Translations/de.lproj/Localizable.strings]: Done 👍  and exactly 30 characters.

File: Translations/de.lproj/Localizable.strings (hunk 12 above)
Comment (hashfunktion) [reply Translations/de.lproj/Localizable.strings]: changed into "Gebe Trinkgeld für Kaffee"

**Review by aonez — APPROVED** [review]

**Comment by aonez** [comment]
Now you've got it!!! Thanks a lot @hashfunktion!

**Comment by hashfunktion** [comment]
@aonez no problem, the limitations are a little bit tricky 😄. Please tell me in the feature if something new to translate in german, than I will help.

**Pull request merged**